EssentialEstablishmentGenerator/NPCGeneration/npcData.js

```diff
@@ -133,9 +591,9 @@ setup.npcData = {
           // Descriptions and stuff goes here
           return [
             'I met my ' + setup.profile(friend, 'best buddy') + ' on some travel.',
-            'I lost contact with an ' + setup.profile(friend, 'old friend') + ', and reconnected with ' + friend.himher + ' on a pilgrimage.',
+            'I lost contact with an ' + setup.profile(friend, 'old friend') + ', and reconnected with ' + setup.profile(friend, friend.himher) + ' on a pilgrimage.',

```

Comment:
Try and only reference the same person once in a paragraph; otherwise, it will make it seem like there are multiple different people.

Reply:
Ah the reason for this was solely because the other outputs I tried for .himher printed out either "undefined" or the literal "$thisNPC.hisher"

Reply:
Ah yeah, we're migrating pronouns and such over to `npc.pronoun` (which is currently `npc.pronouns` but I'll be changing it soon). Maybe it wasn't initialising properly?